cli/jobs/train/r/accident-prediction/accident_r_job.yml

```diff
@@ -0,0 +1,20 @@
+$schema: https://azuremlschemas.azureedge.net/latest/commandJob.schema.json
+description: This job trains a GLM on the accidents dataset to predict fatality 
```

Comment:
please follow order defined in contributing guide for sections of jobs, i.e. description toward the end